src/app/_shared/footer/footer.component.html

```diff
@@ -1,5 +1,5 @@
 <footer class="bg-gray-100 overflow-hidden">
   <div class="max-w-7xl py-6 px-4 sm:px-6 lg:px-8 mx-auto">
-    © 2021 ngMorocco | Angular in darija
+    © {{ new Date().getFullYear() }}  ngMorocco | Angular in darija
```

Comment:
Hey @akiyamaSM :)
Expressions in Angular templates are not JavaScript, they're just inspired from JS. That's only possible with JSX... with its shortcomings.